magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/WebHookActor.scala

```diff
@@ -31,25 +105,38 @@ object WebHookActor {
           this.processAgain = true
         } else {
           this.isProcessing = true
-          println("WebHook Processing: STARTING")
-          processor.sendNotifications().map {
+
+          println(s"WebHook ${this.id} Processing: STARTING")
```

Comment:
Can we use the akka logger for this so it has log levels and is susceptible to setting the log level?